lib/Onyx.js

```diff
@@ -794,6 +802,36 @@ function notifyCollectionSubscribersOnNextTick(key, value) {
     Promise.resolve().then(() => keysChanged(key, value));
 }
 
+/**
+ * This method waits for prior modifications to complete before running the next modification.
+ *
+ * @param {Function} nextModification
+ * @param {String} key If "key" is not set then the modification will be queued after all prior modifications completed.
```

Comment:
If the key might be null this should be a union type I think like `(string|null)`.